src/index.html

```diff
@@ -1,22 +1,380 @@
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link category__link--wide"
+                  >
+                    <img
+                      class="category__photo"
+                      src="images/categoreis/Smart Home 2 - home.png"
+                      alt="Smart Home"
+                    />
+                  </a>
+                </div>
+                <h3 class="category__title">Smart Home</h3>
+              </article>
+            </div>
+          </section>
+        </div>
+      </div>
+    </main>
+
+    <div class="sound"></div>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <section class="aboutus">
+            <h2
+              class="aboutus__title page__section-title section-title--align--left"
+            >
+              Timeless, for 50
+              <br />
+              years and counting
+            </h2>
+            <div class="aboutus__content">
+              <p class="aboutus__description">
+                Bang & Olufsen is expanding its Recreated Classics Program with
+                the launch of Beosystems – a limited edition music system that
+                transcends time by bridging the gap between one of our iconic
+                designs from 1972 and today’s cutting-edge digital technology.
+              </p>
+            </div>
+          </section>
+
+          <section
+            class="contacts"
+            id="contacts"
+          >
+            <h2 class="contacts__title">Contact us</h2>
+            <div class="contacts__section">
+              <div class="contacts__form">
+                <form
+                  class="contacts__form"
+                  onsubmit="this.reset(); return false;"
+                >
+                  <input
+                    type="text"
+                    class="contacts__active contacts__name"
+                    placeholder="Name"
+                    required=""
+                  />
+                  <input
+                    type="email"
+                    class="contacts__active contacts__email"
+                    placeholder="Email"
+                    required=""
+                  />
+                  <textarea
+                    id="message"
+                    class="contacts__active contacts__message"
+                    rows="4"
+                    cols="30"
+                    placeholder="Message"
+                    name="Message"
+                    required=""
+                  ></textarea>
+                  <div class="contacts__button">
+                    <button class="contact__button">Send</button>
+                  </div>
+                </form>
+              </div>
+              <div class="contacts__section--adress">
+                <h4 class="contacts__info-1">Phone</h4>
+                <p class="contact__info-2">+1 234 5555-55-55</p>
```

Comment:
you missed the phone link here:

https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_link_phoneto